dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/export/OrderAndPaginationExporterTest.java

```diff
@@ -1153,6 +1158,96 @@ void shouldOrderEventsByDataElementAndNotFilterOutEventsWithoutThatDataElement()
     assertEquals(List.of("pTzf9KYMk72", "D9PbzJY8bJM"), events);
   }
 
+  private static Stream<Arguments> orderByFieldInDescendingOrderWhenModeSelected() {
+    return Stream.of(
+        Arguments.of("enrollment.uid", "pTzf9KYMk72", "D9PbzJY8bJM"),
+        Arguments.of("enrollment.status", "D9PbzJY8bJM", "pTzf9KYMk72"),
+        Arguments.of("uid", "pTzf9KYMk72", "D9PbzJY8bJM"),
+        Arguments.of("enrollment.enrollmentDate", "D9PbzJY8bJM", "pTzf9KYMk72"));
+  }
+
+  private static Stream<Arguments> orderByFieldInAscendingOrderWhenModeSelected() {
+    return Stream.of(
+        Arguments.of("enrollment.uid", "D9PbzJY8bJM", "pTzf9KYMk72"),
+        Arguments.of("enrollment.status", "pTzf9KYMk72", "D9PbzJY8bJM"),
+        Arguments.of("uid", "D9PbzJY8bJM", "pTzf9KYMk72"),
+        Arguments.of("enrollment.enrollmentDate", "pTzf9KYMk72", "D9PbzJY8bJM"));
+  }
+
+  @ParameterizedTest
+  @MethodSource("orderByFieldInDescendingOrderWhenModeSelected")
+  void shouldOrderEventsByFieldInDescendingOrderWhenModeSelected(
+      String field, String firstEvent, String secondEvent)
+      throws ForbiddenException, BadRequestException {
+
+    eventParamsBuilder.orgUnitMode(SELECTED);
+    eventParamsBuilder.orgUnitUid(orgUnit.getUid());
+    eventParamsBuilder.orderBy(field, SortDirection.DESC);
+
+    List<String> events = getEvents(eventParamsBuilder.build());
+    assertEquals(List.of(firstEvent, secondEvent), events);
+  }
+
+  @ParameterizedTest
+  @MethodSource("orderByFieldInAscendingOrderWhenModeSelected")
+  void shouldOrderEventsByFieldInAscendingOrderWhenModeSelected(
+      String field, String firstEvent, String secondEvent)
+      throws ForbiddenException, BadRequestException {
+
```

Comment:
please stick to the style of the other tests. no newline at the beginning and a newline before and after the method we test.